usecases/MinimumWorkingExample/dodo.py

```diff
@@ -374,3 +386,105 @@ def task_upload_to_openbis():
             'targets': [sample_file_path],
             'clean': [clean_targets],
         }
+
+@create_after(executed='extract_metadata_emodul')
+def task_perform_calibration():
+    """Loop over the experiments and store the calibrated E in csv file. Each iteration generates four files
+    viz. calibrated data.csv, displacement data used, load data used and the knowledge graph file containing the calibration details
+    (these are not specific file names)"""
+    # create folder, if it is not there
+    Path(calibrated_data_directory).mkdir(parents=True, exist_ok=True)
+
+    # defining calibration input, setting the values for the priors
+    E_loc = 30  # KN/mm2 (mean)
+    E_scale = 10  # KN/mm2 (std)
+
+    # setting for fast test, defining the list
+    if config['mode'] == 'cheap':
+        list_exp_name = [single_example_name]
+    else:  # go through all files
+        list_exp_name = os.listdir(
+            processed_data_emodulus_directory)
+        list_exp_name = [os.path.splitext(f)[0] for f in list_exp_name]  # split extension
+    for f in list_exp_name:
+        calibrated_data_expwise = os.path.join(calibrated_data_directory,f) # create new folder for each exp
+        try:
+            # Create the directory
+            os.mkdir(calibrated_data_expwise)
+        except FileExistsError:
```

Comment:
I tracked down the issue on my machine. There is a loop over all files, and then the result directory is newly created. If this directory is already existing, nothing is done at all. However, this leads to several problems when e.g. the computation is stopped during the calibration, because then the directory is existing (thus when rerunning this part is skipped), but the calibrated samples are not existing, thus leading to problems when reading in these samples for the subsequent posterior predictive.  IMO the continue in the  try catch block is the problem and should be removed. I will review the remaining part tomorrow.

Reply:
Check first, if the directory is existing, and if not, create it.

Reply:
updated. please complete the merge so that we can close this.